source/ui.js

```diff
@@ -139,7 +139,7 @@ module.exports = async (options, pkg) => {
 		{
 			type: 'confirm',
 			name: 'publishScoped',
-			when: isScoped(pkg.name) && !options.availability.isAvailable && !options.availability.isUnknown && options.runPublish && (pkg.publishConfig && pkg.publishConfig.access !== 'restricted'),
+			when: isScoped(pkg.name) && options.availability.isAvailable && !options.availability.isUnknown && options.runPublish && (typeof pkg.publishConfig === 'object' && pkg.publishConfig.access !== 'restricted'),
```

Comment:
@itaisteinherz Would mind adding a test case for it and explaining why we should check the type of `pkg.publishConfig`?